assignments/this.js

```diff
@@ -1,26 +1,83 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window/ GLobal Binding:
+*    When in the global scope, the binding will be the windows console. This has a ton of properties that could be accessed.
+*
+* 2. Implicit Binding:
+*   When used inside an object, "this" refers to that object. Most commonly used binding, Whenever a function is called by a preceding dot, the object before that dot is this.
+*
+* 3. New Binding:
+*    Whenever a constructor function is used, this refers to the specific instance of the object that is created and returned by the constructor function.
+*    I would love to descrive new binding in my own words, but I have none.   
+*
+* 4. Explicit Binding:
+*   Used in JavaScripts CALL or APPLY method.
+
 *
 * write out a code example of each explanation above
 */
 
```

Comment:
I feel like you are starting to get a solid handle on the four types of bindings. Great job!